### ISSUE TITLE
What is the SGX demo progress?

### ISSUE THREAD

**Issue by xwang98** [enarx-archive/demo#34]
I am interested in the demo with SGX. After reading some issues, it seems the demo is not working yet? I wonder what is the current plan? thanks!

**Comment by mbestavros** [comment]
@lkatalin is better qualified to talk about this in depth than I am (she wrote it), but in a nutshell --

Our very first SGX demo _is_ working on her fork of the `demo` repository, though it is still being reviewed and tweaked before being merged into the mainline repository.

Regardless, the demo will only work on [an SGX2-capable system](https://github.com/ayeks/SGX-hardware#hardware-with-sgx2-support), of which there are very few -- even if the code does exist and work. I'm not sure if you have one.

**Comment by lkatalin** [comment]
Yes, we are just waiting on some reviews before merging the demo. We can update here as well when it is merged. Thanks for your question.

**Comment by mbestavros** [comment]
The title issue can be tracked at #30. Closing.
